languages/java/exercises/concept/arrays/src/test/java/BirdCountTest.java

```diff
@@ -0,0 +1,77 @@
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.Ignore;
+
+
+import static org.junit.Assert.assertEquals;
+
+public class BirdCountTest {
```

Comment:
We decided to use assert4j instead of junit as assertion library.
Please express the assertions using  assert4j.
You can find an example [here](https://github.com/exercism/v3/blob/master/languages/java/exercises/concept/basics/src/test/java/LasagnaTest.java)